packages/lime-system/files/usr/bin/limed

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/lua
+
+--! LibreMesh community mesh networks meta-firmware
+--!
+--! Copyright (C) 2024  Gioacchino Mazzurco <gio@polymathes.cc>
+--! Copyright (C) 2024  Asociación Civil Altermundi <info@altermundi.net>
+--!
+--! SPDX-License-Identifier: AGPL-3.0-only
+
+local config = require("lime.config")
+local utils = require("lime.utils")
+
+local limed = {};
+
+limed.__PRIVATE_APUP_ENABLED = nil
+
+function limed.APUP_ENABLED()
+	if limed.__PRIVATE_APUP_ENABLED ~= nil then
+		return limed.__PRIVATE_APUP_ENABLED
+	end
+
+	function check_apup_mode_cb(section)
+		if(section['modes'] == null) then return end
+
+		for _,mode in pairs(section['modes']) do
+			if(mode == 'apup') then
+				limed.__PRIVATE_APUP_ENABLED = true;
+				return limed.__PRIVATE_APUP_ENABLED;
+			end
+		end
+	end
+
+	config.foreach(nil, check_apup_mode_cb)
+	return limed.__PRIVATE_APUP_ENABLED
+end
+
+-- No need to run if APuP is not enabled in configuration
+if not limed.APUP_ENABLED() then os.exit(0) end;
+
+local uloop = require("uloop");
+local network = require("lime.network")
+
+local function dumptable(table, nesting)
```

Comment:
This is a duplicate of the function you added in utils.dumptable, right?

Reply:
ups, debugging copy paste leftover